### ISSUE TITLE
Overly long menus need a strategy

### ISSUE THREAD

**Issue by duffyjp** [DFMCH/dfm_web#48]
<img width="1192" alt="screen shot 2018-03-07 at 9 26 28 am" src="https://user-images.githubusercontent.com/382216/37100869-babbebc6-21e9-11e8-8fee-9a613f89de51.png">

**Comment by duffyjp** [comment]
Almost got it.  Added a sprinkle of Javascript to detect long menus and add a "crowded" class.  Then some new CSS to make a compact scrollable menu.  Only problem I've noticed so far is to make it scrollable, I can't use absolute positioning anymore, so the list of links stretches the top menu out making it too wide.

<img width="1176" alt="screen shot 2018-03-07 at 10 41 06 am" src="https://user-images.githubusercontent.com/382216/37105178-246b9fe4-21f4-11e8-9083-2bd5b6ed60b8.png">
